index.js

```diff
@@ -7,3 +7,21 @@ module.exports = array => {
 
 	return array[Math.floor(Math.random() * array.length)];
 };
+
+module.exports.multiple = (array, count) => {
+	if (!Array.isArray(array)) {
+		throw new TypeError('Expected an array');
+	}
+
+	if (!Number.isInteger(count) && count >= 0) {
+		throw new TypeError('Expected a non-negative integer');
+	}
+
+	if (count > array.length) {
+		throw new TypeError(`Expected an array with at least ${count} items`);
+	}
+
+	array = [...array];
+
+	return [...new Array(count)].map(() => array.splice(Math.floor(Math.random() * array.length), 1)[0]);
```

Comment:
Seems like you can do this without splicing (which is very slow).

Reply:
Any reason you're not just calling `module.exports()` multiple times?

Reply:
@sindresorhus Which type of behavior do you want?

Allowing duplicates:

```js
randomItem.multiple([1, 2, 3], 2)
//=> [1, 1]
```

Disallowing duplicates:

```js
randomItem.multiple([1, 2, 3], 2)
//=> [2, 1]
```

Reply:
Allowing duplicates. That would be consistent with the main method. There could potentially be an option to ensure the elements are consecutively unique, and one that ensures the array uses up all elements before starting again.

Reply:
The behavior should be clearly documented.